File: shared/src/commonMain/kotlin/com/test/iliketodo/presentation/ContentViewState.kt
package com.test.iliketodo.presentation

import com.test.DeletedTODOItem
import com.test.TODOItem

data class ContentViewState(
    val itemList: List<TODOItem> = emptyList(),
    val lastDeletedItem: DeletedTODOItem? = null,
    val deletedCount: Int = 0,
    val titleText: String = "",
    val imageUrlText: String = ""
)

File: iosApp/iosApp/ContentViewState.swift
//
//  ContentViewState.swift
//  iosApp
//
//  Created by HYUNGCHAN JUNG on 2023/12/29.
//  Copyright © 2023 orgName. All rights reserved.
//

import shared
import Foundation


struct ContentViewState {
    var itemList: [TODOItem] = []
    var lastDeletedItem: DeletedTODOItem? = nil
    var deletedCount: Int = 0
    var titleText: String = ""
    var imageUrlText: String = ""
}
